be/src/exprs/bitmap_function.cpp

```diff
@@ -388,6 +388,23 @@ BigIntVal BitmapFunctions::bitmap_intersect_finalize(FunctionContext* ctx, const
     return result;
 }
 
+StringVal BitmapFunctions::bitmap_or(FunctionContext* ctx, const StringVal& src, const StringVal& dst){
```

Comment:
src and dst can be null, you should handle this case.

Reply:
OK. I will update doc and fix this case.